src/main/service/listManipulation/CrimeListsService.ts

```diff
@@ -1,8 +1,8 @@
 import moment from 'moment-timezone';
```

Comment:
A few of the methods in the CrimeListService are used by magistrate list too. I wonder should rename the class to CrownListService and not CrimeListService so the name of the class reflects its usage. What do u think?